### PR TITLE
add support for types_check option

### PR DESCRIPTION
**Pull request by r3b-fish**

### CODE REVIEW

File: tests/integration/test_hook.py [hunk 1]
```diff
@@ -34,6 +34,51 @@ def test_get_first(self):
...
+                f'INSERT INTO {self._temp_table_name} (test_field) VALUES',
+                [(date(2012, 10, 25), )],
+                types_check=True,
+            )
+        self.assertIn('Expected UInt8', str(e.exception))
+
+    def tearDown(self) -> None:
+        self._hook.run(
+            f'DROP TABLE IF EXISTS {self._temp_table_name}',
+        )
```
Comment (bryzgaloff) [inline tests/integration/test_hook.py:79]:
```suggestion
        self._hook.run(f'DROP TABLE IF EXISTS {self._temp_table_name}')
```

File: tests/integration/test_hook.py [hunk 2]
```diff
@@ -34,6 +34,51 @@ def test_get_first(self):
...
+        self._hook = ClickHouseHook()
+        self._temp_table_name = f'test_temp_table_for_insert'
+        self._hook.run(
+            f'CREATE TABLE {self._temp_table_name} (test_field UInt8) ENGINE MergeTree() ORDER BY test_field',
+        )
+
+    def test_simple_insert(self):
+        result = self._hook.run(
+            f'INSERT INTO {self._temp_table_name} (test_field) VALUES', [(3,)],
+        )
```
Comment (bryzgaloff) [inline tests/integration/test_hook.py:50]:
```suggestion
        result = self._hook.run(f'INSERT INTO {self._temp_table_name} (test_field) VALUES', [(3,)])
```

**Review by bryzgaloff — CHANGES_REQUESTED** [review]
Good, thank you! Minor style changes required and I will merge and publish.

**Comment by r3b-fish** [comment]
I've done cosmetic changes. Can you check it. Thanks for quick feedback!

**Review by bryzgaloff — APPROVED** [review]

**Comment by bryzgaloff** [comment]
Tough time in Russia, have not had a chance to have a look. Thanks, deploying.

**Comment by bryzgaloff** [comment]
Publishes as https://pypi.org/project/airflow-clickhouse-plugin/0.8.1/

**Pull request merged**